components/combobox/stories/template.js

```diff
@@ -0,0 +1,135 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import { Template as Menu } from '@spectrum-css/menu/stories/template.js';
+import { Template as TextField } from '@spectrum-css/textfield/stories/template.js';
+import { Template as Popover } from '@spectrum-css/popover/stories/template.js';
+import { Template as PickerButton } from '@spectrum-css/pickerbutton/stories/template.js';
+
+import { useArgs, useGlobals } from "@storybook/client-api";
+
+import "../index.css";
+
+export const Template = ({
+  rootClass = "spectrum-Combobox",
+  id,
+  content,
+  customClasses = [],
+  size = "m",
+  isOpen = true,
+  isInvalid = false,
+  isValid = false,
+  isQuiet = false,
+  isDisabled = false,
+  isFocused = false,
+  isKeyboardFocused = false,
+  isLoading = false,
+  isRequired = false,
+  readOnly = false,
+  ...globals
+}) => {
+  const [_, updateArgs] = useArgs();
+  const [{lang}] = useGlobals();
+
+  const { express } = globals;
+
+  try {
+    if (!express) import(/* webpackPrefetch: true */ "../themes/spectrum.css");
+    else import(/* webpackPrefetch: true */ "../themes/express.css");
+  } catch (e) {
+    console.warn(e);
+  }
+
+  return html`
+    <div
+      class=${classMap({
+        [rootClass]: true,
+        [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
+        'is-open': !isDisabled && isOpen,
+        [`${rootClass}--quiet`]: isQuiet,
+        'is-invalid': !isDisabled && isInvalid,
+        'is-valid': !isDisabled && isValid,
```

Comment:
The `Valid` control wasn't doing anything for me. Is it supposed to?

Reply:
I don't think it does anything currently or has any design spec. Though it is listed as a class that should bubble up to the parent in the migration notes.

Reply:
If they're not in the design spec, you can delete it. We can always add back later when the time comes but no need to include it at this time.

Reply:
Valid adds a little green checkmark to the Textfield. Does it maybe need to be passed down to that component?

Reply:
The combobox design spec does not include the valid checkmark.